api/envoy/config/core/v3/extension.proto

```diff
@@ -28,3 +30,30 @@ message TypedExtensionConfig {
   // <config_overview_extension_configuration>` for further details.
   google.protobuf.Any typed_config = 2 [(validate.rules).any = {required: true}];
 }
+
+// Configuration source specifier for a late-bound
+// extension configuration. The parent resource is warmed until all the initial
+// extension configurations are received, unless the flag to apply the default
+// configuration is set. Subsequent extension updates are atomic on a per-worker
+// basis, and apply to new streams while the active streams continue using
+// the older extension configurations. If the initial delivery of the extension
+// configuration fails, due to a timeout for example, the optional default
+// configuration is applied. Without a default configuration, the extension is
+// disabled, until an extension configuration is received.
```

Comment:
As per the other thread I tagged you on, things become somewhat more interesting when the extensions themselves have warming dependencies.

Reply:
I think this means an init manager needs to be re-created for every update and the config swap stalled until its ready. Do you want this in here or is that too much and should have its own issue?

Reply:
Separate issue I think.

Reply:
Filed and added a note.